src/rpc/names.cpp

```diff
@@ -468,6 +473,17 @@ name_show (const JSONRPCRequest& request)
   if (request.params.size () >= 2)
     options = request.params[1].get_obj ();
 
+  /* Parse and interpret the name_show-specific options.  */
+  RPCTypeCheckObj(options,
+      {
+          {"allowExpired", UniValueType(UniValue::VBOOL)},
+      },
+      true, false);
+
+  bool allow_expired = allow_expired_default;
+  if (options.exists("allowExpired"))
+      allow_expired = options["allowExpired"].get_bool();
+
```

Comment:
Please also update the style here as well - space before function call parenthesis, and indentation only two spaces.  The same for the rest of the changes.